### DISCUSSION TITLE
Add Ability to delete Lora, etc from UI interrface.

### DISCUSSION THREAD

**Discussion by kinglunalilo** [Stability-AI/StableSwarmUI#458] [Ideas]
I have a lot of Lora's and love installing new one every day and would love the ability to delete them from the UI as it much quicker and since I'm dealing with a hundred of them literally.

**Comment by mcmonkey4eva** [comment]
Did you miss the readme migration notice? https://github.com/Stability-AI/StableSwarmUI?tab=readme-ov-file#migration-notice
https://github.com/mcmonkeyprojects/SwarmUI/discussions/2
